### ISSUE TITLE
Successfactors : FlexibleColumnLayout in IE11 sometime shows aria-labels as visible

### ISSUE THREAD

**Issue by mydavidzhang** [SAP/ui5-webcomponents#2723]
**Describe the bug**
FlexibleColumnLayout in IE11 sometime shows aria-labels as visible on top right corner when First column is collapsed and only Middle column visible

**To reproduce**
1. Create ``<ui5-flexible-column-layout layout="TwoColumnsMidExpanded"><div slot="startColumn">1</div><div slot="midColumn">2</div></ui5-flexible-column-layout>``
2. Open page on IE11
3. Click arrow icon of flexible column layout to collapse first column
4. Could see aria-labels on top left sometime like overlapped "First Column", "Middle Column" and "Last Column".

**Expected behavior**
aria-labels should NOT be visible

**Screenshots**
![Screenshot 2021-01-22 234243](https://user-images.githubusercontent.com/10460331/105572811-48f6ef00-5d0e-11eb-9afb-539633b6015e.png)
![Screen Shot 2021-01-22 at 11 51 05 PM](https://user-images.githubusercontent.com/10460331/105572812-4a281c00-5d0e-11eb-93b2-defacfaed6b6.png)

**Context**
- UI5 Web Components version : 0.29.0
- OS/Platform: Windows
- Browser: IE11 only
- Other information: Not reproducible in all cases. Can try put different content in startColumn and midColumn like some drop down list. However if one use case can reproduce this issue then could always consistently reproduce it.

**Affected components** 
FlexibleColumnLayout

**Organization:** 
Successfactors

**Comment by vladitasev** [comment]
Hi @mydavidzhang 

I could not reproduce the issue. I tried multiple times with/without cache.

I could imagine something like this could happen on initial load (while CSS is not loaded yet), but never on clicking the arrows.

You could try with this CSS on your HTML page:
```css
.ui5-hidden-text {
	position: absolute;
	clip: rect(1px,1px,1px,1px);
	user-select: none;
	left: 0;
	top: 0;
}
```

Let me know if it helps at all and we'll take it from there.

Regards,
Vladi

**Comment by mydavidzhang** [comment]
Hi Vladi,
This issue happened in IE11 sometime. Looks the .ui5-hidden-text was there for these labels. And after modified and tried that  css in library node_modules/@ui5/webcomponents-fiori/dist/generated/themes/FlexibleColumnLayout.css.js I found the position, user-select, left and top worked when issue happened. Looks the clip css did not work sometimes and that's why the label shows on screen.
To allow screen reader to read these labels suggestion to change top value to negative value like -100rem (or just -2rem), or change both left and top to negative values so that make sure the label is invisible on screen. 
Thanks

**Comment by ilhan007** [comment]
Hello @mydavidzhang thanks for checking. 

We think it's safe to add negative coordinates to ensure the labels are not visible in IE. We will create PR for that.

BR,
ilhan

**Comment by ilhan007** [comment]
Fix is released with 0.29.14
